src/main/views/user-details.njk

```diff
@@ -176,7 +176,7 @@
       </dd>
       <dd class="govuk-summary-list__value">
         <p class="govuk-body-m" id="sso-id">
```

Comment:
There is a copy and paste error in the id here. Think Ashwini is changing this to "createDate" in her branch. Not sure if u ave to change it in your branch too.